pandas/tests/reductions/test_reductions.py

```diff
@@ -558,6 +599,8 @@ def test_minmax_nat_series(self, nat_ser):
         # GH#23282
         assert nat_ser.min() is pd.NaT
         assert nat_ser.max() is pd.NaT
```

Comment:
can parameterize over skipna in a lot of the code you added to tests

Reply:
We just implemented test_reductions, still need to move over the reduction tests from tests.indexes and tests.frames.  After they're all in one place is when to really focus on parametrization.